### DISCUSSION TITLE
Change Icon for Jump to Note button

### DISCUSSION THREAD

**Discussion by chrissy-hi** [TriliumNext/Trilium#6071] [General]
Hey I would like to change the icon of the "Jump To Note" button in the left sidebar. 

The current icon is used to convey that the button will send something. This is not the button's function, so it should be changed to an icon that conveys jumping to.

Does anyone have any suggestions on what icon would be better to convey the button's function? I was thinking of this https://www.shutterstock.com/image-vector/black-share-publish-arrow-social-media-1413531329, but I would love to get community feedback before submitting a pull request.

**Comment by meichthys** [comment]
I'd be interested to see some suggestions too. The one you linked to looks more like a "share" icon to me. 

Perhaps a target icon may work? 
http://icon-sets.iconify.design/bx/target-lock/
